### ISSUE TITLE
Bug: custom cluster theme example not working properly

### ISSUE THREAD

**Issue by timadevelop** [heremaps/maps-api-for-javascript-examples#38]
Steps to reproduce:

1. See example: https://heremaps.github.io/maps-api-for-javascript-examples/custom-cluster-theme/demo.html
2. Click on image

```js
Uncaught (in promise) TypeError: e.target.getPosition is not a function
    at ps.onMarkerClick (demo.js:117)
    at ps.n.gd [as yn] (mapsjs-core.js:61)
```
https://github.com/heremaps/maps-api-for-javascript-examples/blob/8000c44764db4e537e5affa3ad97031817116ba0/custom-cluster-theme/demo.js#L117

**Comment by timadevelop** [comment]
Here is the problem:

```ts
	onMarkerClick(evt) {
		// TODO: BUG:  https://github.com/heremaps/maps-api-for-javascript-examples/issues/38
		let position = evt.target.getPosition() // this line will cause the error
		// Because original target is inside of evt.target.getData()
		const target = evt.target.getData();
		// Works like a charm 
		position = target.getPosition();
		// ...
	}
```

**Comment by kplotnik** [comment]
Fixed in #39
